pkg/api/team_test.go

```diff
@@ -9,11 +9,12 @@ import (
 	"github.com/grafana/grafana/pkg/components/simplejson"
 	"github.com/grafana/grafana/pkg/models"
 
+	"net/http"
```

Comment:
Not sure what the recommendation/standard is, but I have something is VS code that sorts imports when saving a file creating something like this:
```go
// ordered set of builtin Go packages

// ordered set of external Go packages
```

Reply:
@marefr I don't have any control of it, I think it's what the `goimports` formatter does automatically.